tests/_utils/ui.py

```diff
@@ -0,0 +1,115 @@
+# Copyright AS2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import annotations
+
+import sys
+from typing import Iterable
+
+from typing_extensions import TypeAlias
+
+Details: TypeAlias = Iterable[str]
+
+DEFAULT_WIDTH = 60
+
+try:
+    import colorama  # type: ignore[import]
```

Comment:
Another proposal: I would like to add `colorama` (available on both conda and pip) as a dev dependency to the env files. However, I did make it so that there will be clean output even if it is missing.

Reply:
sounds like a good idea. feel free to update the conda environment accordingly and also let @marcinz update the script to generate the ci image whenever you can.

Reply:
and in this PR: https://github.com/nv-legate/cunumeric/pull/378/commits/40518c6dfa7e7681a64d7122584934451cb4ac8f